dali/benchmark/operator_bench.h

```diff
@@ -29,29 +29,33 @@ class OperatorBench : public DALIBenchmark {
   void RunCPU(benchmark::State& st, OpSpec op_spec,
               int batch_size = 128,
               int H = 1080, int W = 1920, int C = 3,
-              bool fill_in_data = false) {
+              bool fill_in_data = false, int num_threads = 4) {
```

Comment:
is 4 threads the default case now?

Reply:
I saw it used somewhere in our samples, so assumed that this is a good default for a benchmark.